src/tickit/core/management/schedulers/base.py

```diff
@@ -69,24 +70,37 @@ async def update_component(self, input: Input) -> None:
         """
         await self.state_producer.produce(input_topic(input.target), input)
 
+    async def skip_component(self, skip: Skip) -> None:
+        """Sends a message to itself that a given component update has been skipped.
+
+        Args:
+            skip (Skip): The Skip information to be included in the message sent to the
+                scheduler.
+        """
+        await self.state_producer.produce(output_topic(skip.source), skip)
+
     async def handle_message(self, message: ComponentOutput) -> None:
         """Handle messages received by the state consumer.
 
-        An asynchronous callback which handles Interrupt, Output and ComponentException
-        messages received by the state consumer. For Outputs, changes are propagated
-        and wakeups scheduled if required. For interrupts handling is deferred. For
-        exceptions, a StopComponent message is produced to each component in the system
-        to facilitate shut down.
+        An asynchronous callback which handles Interrupt, Output, ComponentException
+        and Skip messages received by the state consumer. For Outputs, changes are
+        propagated and wakeups scheduled if required. Skips are also propgated. For
+        interrupts handling is deferred. For exceptions, a StopComponent message is
+        produced to each component in the system to facilitate shut down.
 
         Args:
-            message (Union[Interrupt, Output, ComponentException]): An Interrupt,
+            message (Union[Interrupt, Output, ComponentException, Skip]): An Interrupt,
```

Comment:
What was there initially should instead be the defined type alias `ComponentOutput`. However, I intentionally left skip out of it because it doesn't nicely fit in the `Input` or `Output` message types, since they are not sent like the other messages. I felt it prudent to keep the differentiation.

Reply:
@callumforrester No, nothing breaks per say, its just the `Google` doc string convention which we have kept to for the rest of the code base, so I'm reluctant to change it in one place.

Reply:
What should have been flagged, is the function signature has the wrong type in, so I'll fix that now.